examples/benchmarks/GeneralPtNN/README.md

```diff
@@ -0,0 +1,15 @@
+
+
+# Introduction
+
+What is GeneralPtNN
+- Fix previous design that fail to support both Time-series and tabular data
+- Now you can just replace the Pytorch model structure to run a NN model.
+
+We provide an example to demonstrate the effectiveness of the current design.
+- `workflow_config_gru.yaml` align with previous results [GRU(Kyunghyun Cho, et al.)](../README.md#Alpha158 dataset)
```

Comment:
The links do not work
![image](https://github.com/microsoft/qlib/assets/465606/054994d2-5fa2-4190-8331-eaecf9626d64)